posthog/temporal/workflows/batch_exports.py

```diff
@@ -43,27 +46,39 @@ async def get_rows_count(client, team_id: int, interval_start: str, interval_end
     return int(count)
 
 
+FIELDS = """
+DISTINCT ON (event, cityHash64(distinct_id), cityHash64(uuid))
+toString(uuid) as uuid,
+team_id,
+timestamp,
+inserted_at,
+created_at,
+event,
+properties,
+-- Point in time identity fields
+toString(distinct_id) as distinct_id,
+toString(person_id) as person_id,
+person_properties,
+-- Autocapture fields
+elements_chain
+"""
+
+
 def get_results_iterator(
-    client, team_id: int, interval_start: str, interval_end: str
+    client, team_id: int, interval_start: str, interval_end: str, legacy: bool = False
```

Comment:
Everywhere same default
```suggestion
    client, team_id: int, interval_start: str, interval_end: str, legacy: bool = True
```